### ISSUE TITLE
Is there a way to internationalize URLs?

### ISSUE THREAD

**Issue by sornii** [kadirahq/flow-router#723]
I might use it and I'm thinking how to do it.

Anyone there knows how to use i18n to URL?

Like:

sornii.com/about (english)
sornii.com/sobre (portuguese)
sornii.com/chi-siamo (italian)

**Comment by gurjit03** [comment]
@Sornii did you figured it out ?

**Comment by sornii** [comment]
Nah, I think it should be better to build a route generator for flow-router with a JSON containing an key-value object with value being an array of possibilities using together with Tap i18n for URL links.
